docs/best_practices_guide.adoc

```diff
@@ -421,7 +421,7 @@ For consistency, role names SHOULD follow the above naming pattern. It is import
 Many times the `technology` portion of the pattern will line up with a package name. It is advised that whenever possible, the package name should be used.
 
 .Examples:
-* The role to configure an OpenShift Master is called `openshift_master`
+* The role to configure a Master is called `openshift_master`
```

Comment:
s/a Master/a master/

nitpicky, but not sure master should be capitalized in this context...  probably shouldn't have been in the first place :)